### DISCUSSION TITLE
MSW로 API Mocking으로 개발환경을 높이고 Next.js RSC환경에서도 MSW 적용하기 | 한동룡 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [eastdraconis/blog#2] [Announcements]
# MSW로 API Mocking으로 개발환경을 높이고 Next.js RSC환경에서도 MSW 적용하기 | 한동룡 기술 블로그

MSW로 API Mocking을 하여 프론트엔드 개발 환경을 개선하고 Next.js의 App router의 RSC환경에서 MSW를 적용하는 방법을 알아봅니다.

https://handongryong.com/posts/msw

**Comment by sni424** [comment]
감사합니다. 
다른 블로그들이 이전 버전이라 애먹었는데
작성해주신 글 보고 하니 잘나오네요.

**Reply by eastdraconis** [reply]
도움이 되었다니 다행이네요☺️
